content/en/docs/ops/ambient/getting-started/test.sh

```diff
@@ -75,7 +75,6 @@ _verify_contains snip_l4_authorization_policy_3 "$snip_l4_authorization_policy_3
 _verify_failure snip_l4_authorization_policy_4
 
 _verify_contains snip_l7_authorization_policy_1 "$snip_l7_authorization_policy_1_out"
-_verify_contains snip_l7_authorization_policy_2 "Resource programmed, assigned to service"
```

Comment:
Why this is being removed?

Reply:
I removed it because that step was checking that the bookinfo gateway was programmed.
It felt it was no longer needed considering that istioctl will now perform that check instead.

In retrospect, there is still usefulness in preserving it, even if it is only to confirm that the command we describe in the md functions properly, no matter how redundant that step may seem.

The alternative would be to remove the following text from the markdown itself, but I feel it gives useful context to the reader on what is going on.

```
View the `productpage` waypoint proxy status; you should see the details of the gateway
resource with `Programmed` status:

{{< text bash >}}
$ kubectl get gtw bookinfo-productpage -o yaml
...
```

I will made a commit to restore it

Reply:
Had no preference but was curious. Thanks for the information!